src/app/components/LinkData/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import { string } from 'prop-types';
+
+const LinkData = ({
```

Comment:
Just noticed that the component is called `LinkData` - can you please update it to be `LinkedData` (in the filenames as well as component names) This is since Linked Data is the name of the format we're using here. https://json-ld.org/